testing/test_cacheREADME.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import, division, print_function
+import _pytest
+
+pytest_plugins = ("pytester",)
+
+
+class Helper(object):
```

Comment:
I don't think we need this class at all, it only adds to (albeit small) increase in code complexity without any benefit.

I suggest to just move the `check_readme` function to `TestHelper` and delete this class.